packages/a11y-base/src/tabindex-mixin.js

```diff
@@ -70,9 +74,37 @@ export const TabindexMixin = (superclass) =>
      * @protected
      */
     _tabindexChanged(tabindex) {
+      if (this._shouldAllowFocusWhenDisabled()) {
+        return;
+      }
```

Comment:
Could test new `tabindex-mixin` logic in `a11y-base/test/tabindex-mixin.test.js`

Reply:
This is now a private method.